web/source/build.sh

```diff
@@ -170,6 +207,8 @@ if [ $BUILD_EMBED = true ]; then
 
     rm $EMBED_OUTPUT/keyman.js 2>/dev/null
```

Comment:
Maybe add
`echo Compile embedded...` to differentiate 
`echo Typescript compiled` in lines 210 and 234